### DISCUSSION TITLE
关于目录格式调整

### DISCUSSION THREAD

**Discussion by encounter5639** [sjtug/SJTUThesis#963] [展示区]
使用sjtuthesis模板和使用官方latex模板的小伙伴在审核过程中均收到了调整目录格式的反馈，使用sjtuthesis时收到的反馈具体如下：“目录格式需要微调，未包含摘要、ABSTRACT、插图、表格等。插图、表格一般是排在参考文献之后的。章目录格式：宋体四号,单倍行距，页码右对齐。下划线点按照模板样式调整。小节目录：宋体小四号，单倍行距，左缩进1个汉字符，两端对齐，页码右对齐。二级小节目录：宋体五号,单倍行距，左缩进2个汉字符，两端对齐，页码右对齐”。目前存在的问题主要是①一级、二级、三级目录字体大小变化不符合要求，三级目录左侧缩进过大②插图、表格移至参考文献之后，目录中不再显示插图、表格③下划线点与页码数字存在空隙，官方word模板无空隙。

**Comment by LogCreative** [comment]
不同审核老师会给出不同的参考意见。参考解决方案：
① 一般只需要更改多级目录缩进：
```latex
\setlength{\cftsecindent}{1em}
\setlength{\cftsubsecindent}{2em}
```
② 建议直接去除插图、表格、算法索引。
③ 参照 https://github.com/sjtug/SJTUThesis/discussions/917 改成更小的间距。

**Reply by hotpot-fish** [reply]
现在模板的一级、二级目录字号大小好像没有区别，但图书管理员意见是章目录、小节目录、二级小节目录字号逐渐缩小，有办法调整吗？

**Reply by LogCreative** [reply]
关注每条意见的第一句话，如果只需要调整间距可以认为不需要调整。如果第一句话需要字号逐级减少，请展示一下该意见。

**Reply by hotpot-fish** [reply]
嗯嗯，这是我收到的意见截图
![image](https://github.com/sjtug/SJTUThesis/assets/74254099/2c7a6609-21f7-4346-b024-4a495ecb434f)

**Reply by hotpot-fish** [reply]
> 关注每条意见的第一句话，如果只需要调整间距可以认为不需要调整。如果第一句话需要字号逐级减少，请展示一下该意见。

还有我认识的其他同学收到类似更详细的意见
<img width="551" alt="01d93366abcf60d15e6b2a10c8620a1" src="https://github.com/sjtug/SJTUThesis/assets/74254099/64ac9915-f45b-4cbb-95b2-2970ae399053">

**Reply by LogCreative** [reply]
实际上即使是教务处官方模板也没有实现这一点，这边建议直接跟图书馆审核专家沟通...

**Reply by LogCreative** [reply]
沟通时声明使用的是 LaTeX 模板。

**Reply by Yuan-Allen** [reply]
我的审核意见也提到了这个问题，并且在此之前已经修改过目录缩进了。请问如果要修改目录字号的话要如何修改呢？
<img width="663" alt="image" src="https://github.com/sjtug/SJTUThesis/assets/46676827/6e68ef79-ce8b-4d23-8957-b195ffe04bd6">

**Reply by Yuan-Allen** [reply]
已通过
```Latex
\renewcommand{\cftchapfont}{\zihao{4}\bfseries}
\renewcommand{\cftsecfont}{\zihao{-4}}
\renewcommand{\cftsubsecfont}{\zihao{5}}
```
解决
